config-generators/cosmosdb_nosql-commands.txt

```diff
@@ -12,4 +12,4 @@ add Earth --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.earth" -
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:create" --fields.include "id" --fields.exclude "name"
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.include "id,type" --fields.exclude "name"
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:update" --fields.exclude "*"
-update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "authenticated:create,read,update,delete"
```

Comment:
maybe it's just my eyes here, but I don't see a change between these two lines, is it some hidden character or line ending being altered?

Reply:
There shold be a new long character at the end of Line 15 so that on macOS/Linux it reads Line 15.